app/models/JobExecution.java

```diff
@@ -58,6 +58,7 @@
     public static final String jobExecUrl = "jobExecUrl";
     public static final String jobDefinition = "jobDefinition";
     public static final String flowExecution = "flowExecution";
+    public static final String autoTuningFault = "autoTuningFault";
```

Comment:
All auto tuning related parameters are part of TuningJobExecution. This should be part of TuningJobExecution.

Reply:
Forgot about deprecation of earlier table tuning_job_execution. Let's keep it in job_execution only.